.azure/pipelines/jobs/default-build.yml

```diff
@@ -115,20 +120,25 @@ jobs:
   steps:
   - checkout: self
     clean: true
+  - ${{ if eq(parameters.agentOs, 'Windows') }}:
+    - powershell: ./eng/scripts/InstallVisualStudio.ps1 -Edition BuildTools -Quiet
+      displayName: Install Build Tools for Visual Studio 2019
   - ${{ if eq(parameters.installNodeJs, 'true') }}:
     - task: NodeTool@0
       displayName: Install Node 10.x
       inputs:
         versionSpec: 10.x
   - ${{ if and(eq(parameters.installJdk, 'true'), eq(parameters.agentOs, 'Windows')) }}:
-    - powershell: ./eng/scripts/InstallJdk.ps1 '11.0.1'
+    - powershell: |
+        ./eng/scripts/InstallJdk.ps1 '11.0.1'
+        Write-Host "##vso[task.prependpath]$env:JAVA_HOME\bin"
```

Comment:
Not quite sure the reasoning for moving all of this around. Seems like it's less verbose now.

Reply:
Just cleaning up some duplication and shortening this build. The `|` combines the following lines into a single build step.